com.hybris.hyeclipse.utils/build.properties

```diff
@@ -0,0 +1,9 @@
+ source.. = src/
+output.. = bin/
+bin.includes = plugin.xml,\
```

Comment:
you don't use any extension points in this project, I  would get rid of plugin.xml